src/UI/Seller/src/assets/appConfigs/defaultadmin-test.json

```diff
@@ -1,12 +1,12 @@
 {
   "hostedApp": true,
   "sellerID": "my-seller-id",
-  "sellerName": "Default Admin",
-  "appname": "Default Admin",
-  "clientID": "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx",
-  "middlewareUrl": "https://my-default-admin-test.com",
+  "sellerName": "Headstart Demo",
+  "appname": "Headstart Admin",
+  "clientID": "3330614E-D468-4694-AE87-23DD53872E83",
+  "middlewareUrl": "https://headstartdemo-middleware-test.azurewebsites.net",
   "cmsUrl": "https://ordercloud-cms-test.azurewebsites.net",
-  "translateBlobUrl": "https://my-translation-files/ngx-translate/i18n/",
-  "blobStorageUrl": "https://myblobstoragecontainer.blob.core.windows.net",
+  "translateBlobUrl": "https://headstartdemo.blob.core.windows.net/ngx-translate/i18n/",
+  "blobStorageUrl": "https://headstartdemo.blob.core.windows.net",
```

Comment:
Are these meant to be committed?

Reply:
No, sorry. I always forget to remove them